Tests/MusixmatchAPITests/MusixmatchAPITests.swift

```diff
@@ -10,7 +10,7 @@ import XCTest
 
 final class MusixmatchAPITests: XCTestCase {
 
-    func test_search_withArtistNamesAndTitle_getSongLyrics() async throws {
+    func test_trackSearch_withArtistNamesAndTitle_getSongLyrics() async throws {
```

Comment:
Add edge cases like:
- wrong combination of artist and track
- latest/nonexistent track that hasn't been searchable